apps/api/src/app/admin/admin.service.ts

```diff
@@ -646,6 +649,24 @@ export class AdminService {
     return { marketData, count: marketData.length };
   }
 
+  private async countUsersWithAnalytics(): Promise<number> {
```

Comment:
It can be but, this method being the uppermost private method, linter will throw warning that all the private methods needs to be put below public methods. what should I do @dtslvr

Reply:
No, there is `private async getMarketDataForCurrencies(): Promise<AdminMarketData> {` above... 🙂

Reply:
Done.